docs/how-to-write-a-userstyle.md

```diff
@@ -0,0 +1,208 @@
+<p align="center">
+  <h2 align="center">📖 How to write a userstyle</h2>
+</p>
+
+<p align="center">
+	Beginner-friendly guide for creating a userstyle!
+</p>
+
+&nbsp;
+
+### Prerequisites
+
+- [Stylus](https://github.com/openstyles/stylus) ([Firefox](https://addons.mozilla.org/en-GB/firefox/addon/styl-us/) / [Chrome](https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne)
+  extension).
+- An IDE (e.g. [Visual Studio Code](https://code.visualstudio.com/))
+- A website to theme!
+
+### Setup
+
+Assuming you have completed steps 1-6 of [Creating Userstyles](./userstyle-creation.md#creating-userstyles), you should have the template somewhat filled out. Follow along with your userstyle as we write one for [example.org](http://example.org/).
```

Comment:
Hmm I probably don't need this since we walk through this part in the next step...

Reply:
Well steps 1-4 *are* relevant. Maybe I could literally go through each and every step on that list (https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md#creating-userstyles)?

That would mean adding sections on adding a new userstyle to `userstyles.yml` and creating image previews (this is an important one - we should walk through taking good screenshots, converting to WEBP, and of course using catwalk).

Reply:
I thinks its alright, just remove "steps 1-6 of"